piptools/scripts/sync.py

```diff
@@ -107,6 +108,17 @@ def cli(
             log.error("ERROR: " + msg)
             sys.exit(2)
 
+    # include the wheels requirements txt if we find any files with `_wheels`
+    # suffix
+    wheel_req_files = set()
+    for src_file in src_files:
+        name, ext = os.path.splitext(src_file)
+        wheel_src = name + WHEELS_REQUIREMENTS_SUFFIX + ext
```

Comment:
maybe worth creating a util function for this.  It could be used above as well.

Reply:
yeah I created one in `utils` as `format_suffixed_path`